etc/pr_check/main.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/google/go-github/v54/github"
+	flag "github.com/spf13/pflag"
```

Comment:
I don't think we should alias imports except to resolve duplicates or where the path name isn't the package name.  i.e. including src/server/pps and src/pps in the same module, or importing something like go-foobar where foobar is the actual package name.  (It's not necessary to alias the second case, but goland's copy of goimports does it, so it ends up in the codebase regardless.)